source/fundamentals/auth.txt

```diff
@@ -87,18 +89,18 @@ mechanism:
       string, omit the mechanism. Your code to instantiate a ``MongoClient``
       should look something like this:
 
-      .. code-block:: java
-
-         MongoClient mongoClient = MongoClients.create("mongodb://<username>:<password>@<hostname>:<port>/?authSource=<authenticationDb>");
-
+      .. literalinclude:: /examples/generated/AuthTest.snippet.default-cred-string.kt
+         :language: kotlin
+      
    .. tab::
       :tabid: MongoCredential
 
       To specify the default authentication mechanism using the
       ``MongoCredential`` class, use the ``createCredential()`` method. Your
       code to instantiate a ``MongoClient`` should look something like this:
 
-      .. include:: /includes/fundamentals/code-snippets/auth-credentials-default.rst
+      .. literalinclude:: /examples/generated/AuthTest.snippet.default-mongo-cred.kt
+         :language: kotlin
 
 For more information on the challenge-response (CR) and salted
 challenge-response authentication mechanisms (SCRAM) that MongoDB supports,
```

Comment:
Can't comment, but in the line below change server manual to Server manual

Reply:
Updated